tests/sources/test_cds.py

```diff
@@ -102,6 +102,24 @@ def test_cds_netcdf_selection_limited():
     assert len(s) == 9
 
 
+@pytest.mark.long_test
```

Comment:
A similar test should be added to `ecmwf_api`

Reply:
As discussed, tests added for other formats from the CDS (grib and netcdf).